### ISSUE TITLE
Transform GitHub fork into source

### ISSUE THREAD

**Issue by hongaar** [nabble/unveil2#16]
For better findability.

What do you think?

**Comment by michaelx** [comment]
If you’ve changed a lot compared to the original source, yes.

In any case I’d appreciate a changelog, which should help to bring people over from the inactive unveil.

**Comment by hongaar** [comment]
Yeah, that's a good point. Will make sure we have that with version 3.

**Comment by hongaar** [comment]
Created #17 to track creation of changelog / migration guide.

I plan to release a next minor version as **v2.1.0** with a couple more features (i.e. srcset support) but I think we should move away from the **unveil2** name as soon as possible, as this will create confusion as soon as we arrive at a next major release.

Name suggestions for the new repo are welcome!
